docker-compose.yaml

```diff
@@ -96,6 +97,7 @@ services:
       UI: ${UI:-false}
       UI_PORT: ${UI_PORT:-5005}
       CLI_ENDPOINT: ${CLI_ENDPOINT:-4001}
+      TORCHVISION: "no"
```

Comment:
Is it necessary to have this environment variable because you define it inside the dockerfile